tracker.rst

```diff
@@ -7,23 +7,32 @@ Issue Tracking
 Using the Issue Tracker
 =======================
 
-If you think you found a bug in Python, you can report it to the
-`issue tracker`_.  Documentation bugs can also be reported there.
-Issues about the tracker should be reported to the `meta tracker`_.
+If you think you have found a bug in Python, you can report it to the
+`issue tracker`_. The `issue tracker`_ is also commonly referred to as 
+`bugs.python.org` and `bpo`.  Documentation bugs can also be reported there.
+
+You can report bugs with how the issue tracker functions to the `meta tracker`_.
```

Comment:
For some reason, I don't find the line about the meta tracker easy to read.  First time through, I had read it twice to understand what was being said.  Maybe 'You can report bugs with the issue tracker itself to the meta tracker.'?

Reply:
Thanks for the suggestion. Will update.